endpoints/openrtb2/auction.go

```diff
@@ -1653,3 +1654,23 @@ func checkIfAppRequest(request []byte) (bool, error) {
 	}
 	return false, nil
 }
+
+func getIntegrationTypeFromRequest(req *openrtb_ext.RequestWrapper) (string, error) {
+	reqExt, err := req.GetRequestExt()
+	if err != nil {
+		return "", err
+	}
+	reqPrebid := reqExt.GetPrebid()
+	return reqPrebid.IntegrationType, nil
+}
+
+func validateIntegrationType(req *openrtb_ext.RequestWrapper, integrationTypes map[string]bool) error {
+	integrationType, err := getIntegrationTypeFromRequest(req)
+	if err != nil {
+		return err
+	}
+	if _, ok := integrationTypes[integrationType]; ok {
+		return nil
+	}
```

Comment:
If the integration has not been specified in the request, I believe that's valid according to the corresponding issue, is that correct? It looks like the integration would be set to some default value specified in the config in this case. If I have that right, you'll want to handle that case here as I don't imagine specifying an empty string or nil value in the slice of valid integration values in the config.

Reply:
You're right in bringing up this point! However, I'll handle this Default Integration in a future PR, since we're breaking up my implementation of Integration Type and Channel into multiple smaller PRs.